Assets/Scripts/Controllers/MouseController.cs

```diff
@@ -369,6 +396,26 @@ void UpdateDragging()
                             {
                                 isValid = true;
                             } 
+                        } else if (dragType == "path")
```

Comment:
Oops, it got merged while I was checking the code.

That could be an option, although I am not sure how many more ways to drag there is in grid system.

I would suggest to remove the restriction of only allowing drag for 1x1 furniture, and make a "none" dragType which would be the default for all the furniture. In the future if some would want to make a 2x2 big pipe furniture, then it would work with dragging.